clone_all.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Utility script to clone all Quiver repos into sibling directories of the core
+# quiver repo. Adds origin (user-specific fork) and upstream remotes.
+
+set -e
+
+function _usage {
+  echo "usage: clone_all.sh github_user"
+}
+
+github_user=$1
+
+if [[ -z $github_user ]]; then
+  _usage
+  exit 1
+fi
+
+REPOS="quiver_cache \
+  quiver_check \
+  quiver_clock \
+  quiver_collection \
+  quiver_hashcode \
+  quiver_io \
+  quiver_iterables \
+  quiver_mirrors \
+  quiver_pattern \
+  quiver_strings \
+  quiver_time"
+
+cd ..
+for repo in $REPOS; do
+  if [[ -e $repo ]]; then
+    echo "INFO: $repo already exists. Skipping."
+  else
+    git clone git@github.com:$github_user/$repo.git
```

Comment:
Separate issue: what are your thoughts on cloning the _user_ repo vs cloning the _upstream_ repo? I actually suspect that it's more useful to clone the upstream repo so you've got the latest then edit the remotes to point:
origin -> user repo
upstream -> QuiverDart repo